engines/director/director.cpp

```diff
@@ -79,12 +79,60 @@ DirectorEngine::DirectorEngine(OSystem *syst, const DirectorGameDescription *gam
 
 	SearchMan.addSubDirectoryMatching(_gameDataDir, "data");
 	SearchMan.addSubDirectoryMatching(_gameDataDir, "install");
-	SearchMan.addSubDirectoryMatching(_gameDataDir, "main");		// Meet Mediaband
+	SearchMan.addSubDirectoryMatching(_gameDataDir, "main");							// Meet Mediaband
 	SearchMan.addSubDirectoryMatching(_gameDataDir, "l_zone");
-	SearchMan.addSubDirectoryMatching(_gameDataDir, "win_data", 0, 2);	// L-ZONE
+	SearchMan.addSubDirectoryMatching(_gameDataDir, "win_data", 0, 2);					// L-ZONE
```

Comment:
Ugh, this is now too ugly. Could you instead expose the list from detection.cpp file, then have a smaller list here with dirs that require depth and iterate over it?

Reply:
Done. Now I need to fix the existing subdirectories based on our Discord discussion and my incorrect understanding of the depth parameter.

Reply:
This is now fixed with the latest commit.